Library/Homebrew/dev-cmd/livecheck.rb

```diff
@@ -48,28 +48,28 @@ def livecheck
       puts ENV["HOMEBREW_LIVECHECK_WATCHLIST"] if ENV["HOMEBREW_LIVECHECK_WATCHLIST"].present?
     end
 
-    formulae_to_check = if args.tap
-      Tap.fetch(args.tap).formula_names.map { |name| Formula[name] }
+    formulae_and_casks_to_check = if args.tap
```

Comment:
Depending on the context, this is technically formulae and/or casks to check. Considering the verbosity as well, how would be feel about a more generic name like `items_to_check`?

Reply:
`items_to_check` or `recipes_to_check`?